### PR TITLE
add new contributors to the  readme file

### PR DESCRIPTION
**Pull request by leogermani**
Add new contributors to the Readme file

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
+Contributors: automattic, adamkheckler, aduth, akirk, allendav, alternatekev, andy, annezazu, apeatling, azaozz, batmoo, barry, beaulebens, blobaugh, brbrr, cainm, cena, cfinke, chaselivingston, chellycat, clickysteve, csonnek, danielbachhuber, davoraltman, daniloercoli, delawski, designsimply, dllh, drawmyface, dsmart, dzver, ebinnion, egregor, eliorivero, enej, eoigal, erania-pinnera, ethitter, gcorne, georgestephanis, gibrown, goldsounds, hew, hugobaeta, hypertextranch, iammattthomas, iandunn, jblz, jasmussen, jeffgolenski, jeherve, jenhooks, jenia, jessefriedman, jgs, jkudish, jmdodd, joanrho, johnjamesjacoby, jshreve, kbrownkd, keoshi, koke, kraftbj, lancewillett, lschuyler, macmanx, martinremy, matt, matveb, mattwiebe, maverick3x6, mcsf, mdawaffe, MichaelArestad, migueluy, mikeyarce, mkaz, nancythanki, nickmomrik, obenland, oskosk, pento, professor44, rachelsquirrel, rdcoll, ryancowles, richardmuscat, richardmtl, roccotripaldi, samhotchkiss, scarstocea, scottsweb, sdquirk, simison, stephdau, tmoorewp, tyxla, Viper007Bond, westi, yoavf, zinigor, mdbitz, fgiannar, sermitr, biskobe, leogermani, kbrownkd
```
Comment (jeherve) [inline readme.txt:2]: Do you think you could add them alphabetically, so they're easier to find in the future and to avoid duplicates?

**Review by jeherve — CHANGES_REQUESTED** [review]
Good call. Maybe that's an opportunity to update https://github.com/Automattic/jetpack/blob/050161b9bb1e1630913dd1a33bc62cc0a6bb1da9/readme.md#L54 too?

File: readme.txt (hunk 1 above)
Comment (leogermani) [reply readme.txt]: sure! missed that

**Comment by leogermani** [comment]
> Good call. Maybe that's an opportunity to update
> 
> https://github.com/Automattic/jetpack/blob/050161b9bb1e1630913dd1a33bc62cc0a6bb1da9/readme.md#L54
> too?

done!

**Review by jeherve — APPROVED** [review]

**Pull request merged**